index.html

```diff
@@ -865,10 +919,86 @@ <h4>issuance</h4>
 <section id="conformance"></section>
 
 <section>
-<h2>Privacy Considerations</h2>
+  <h2>Privacy Considerations</h2>
+  <p>
+    Verifiable Credentials often contain sensitive information that should be protected 
+    to ensure the privacy and security of individuals. This section outlines some privacy 
```

Comment:
```suggestion
    to ensure the privacy and security of organizations and individuals. This section outlines some privacy 
```

Reply:
Might want to find a way to get `confidentiality` in here, as it is more relevant to organization use case.